Thunder dome

jfc978 opened the pull request:
Initial creation and testing of Thunder Dome. Generates infinite waves of predetermined enemies.
Press J to enter Thunderdome

Commit:
ThunderDome Initial Push

Press J to enter ThunderDome. No current exit, just has a set number of enemies than eventually errors.

Commit:
Infinite Waves

Never die

bhasseman reviewed (approved):
A LOT of good cleanup in here on top of the Thunderdome implementation. Looks very good, everything builds and works correctly, pressing J takes Link into the thunderdome which functions correctly (spawning waves of enemies, giving a short cooldown between waves, cooldown begins after all enemies are killed). Should get this merged ASAP so we can continue its implementation and turn it into the end game state with victory conditions.
LGTM, well done

royjpark commented:
This looks sick, nice Game1 cleanup. Everything works as expected, LGTM.

rustbeltdurt committed:
Standardized Thunderdome room.

rustbeltdurt committed:
Quick fix to Room Indexing problem on  minimap

Commit:
Key Debouncing

Handle all sensitive keys via debouncing. Sword, music, and secondary attacks.
Also Link now starts with only one key

Commit:
Inventory Health Fix

rustbeltdurt committed:
Added key & chest for triforce.

bhasseman committed:
Moved NPCs

All were moved to the right to make better use of the room, and the skeletons were spread out a bit more.

rustbeltdurt committed:
Merge branch 'ThunderDome'. oop

rustbeltdurt committed:
Merge branch 'ThunderDome' again

rustbeltdurt committed:
On a roll with these.

Commit:
Thunderdome Labels

jfc978 commented:
A lot of improvements and progress on Thunderdome. Merging to reduce clutter and issues

jfc978 committed:
Merge branch 'master' into ThunderDome

Commit:
Fix Boss Constructor
